R/pkg/R/generics.R

```diff
@@ -395,6 +396,9 @@ setGeneric("value", function(bcast) { standardGeneric("value") })
 
 ####################  SparkDataFrame Methods ########################
 
+#' @param x a SparkDataFrame or GroupedData.
```

Comment:
I was thinking if in the future we want to add other methods associated with that generic, would it be more acceptable to change the doc of the generic than the doc of another associated method (seemingly parallel to the added ones)?